apis/apps/v1alpha1/cluster_types.go

```diff
@@ -600,6 +608,31 @@ type ClusterNetwork struct {
 	PubliclyAccessible bool `json:"publiclyAccessible,omitempty"`
 }
 
+type ServiceRef struct {
+	// name of the predefined service reference. references the serviceRefDeclaration name defined in clusterDefinition.componentDefs[*].serviceRefDeclarations[*].name
+	// +kubebuilder:validation:Required
+	Name string `json:"name"`
+
+	// When referencing a service provided by other KubeBlocks cluster, you need to provide the name of the Cluster being referenced. and the referenced cluster should be in the same namespace.
+	// By default, when other KubeBlocks Cluster are referenced, the ClusterDefinition.spec.connectionCredential secret corresponding to the referenced Cluster will be used to bind to the current component.
```

Comment:
ClusterDefinition.spec.connectionCredential should not be the default credential to use, it has too high privileges for external access to the service.

Reply:
> ClusterDefinition.spec.connectionCredential should not be the default credential to use, it has too high privileges for external access to the service.

If ClusterDefinition.spec.connectionCredential is not used, and there is no available connectionCredential at the moment, requiring users to create one separately would impose additional usage burden. We need to further examine the solution for this situation.